core/src/HIP/Kokkos_HIP_Parallel_Range.hpp

```diff
@@ -121,6 +121,7 @@ class ParallelReduce<CombinedFunctorReducerType, Kokkos::RangePolicy<Traits...>,
   using value_type     = typename ReducerType::value_type;
   using reference_type = typename ReducerType::reference_type;
   using functor_type   = FunctorType;
+  using reducer_type   = ReducerType;
```

Comment:
was this added becuase needed for the graph?

Reply:
yes